docs/ssr.md

```diff
@@ -69,3 +69,44 @@ module.exports = {
   plugins: [...otherGatsbyPlugins, 'gatsby-plugin-emotion']
 }
 ```
+
+## Puppeteer
+
+If you are using Puppeteer to prerender your application, you have to disable emotion's `speedy` option to fully render all needed CSS inside the DOM.
+
+index.js
+
+```js
+import { sheet } from 'emotion'
+
+import mount from './mount'
+
```

Comment:
This still won't work since `./mount` will run before speedy is disabled. It needs to be something like this. (though replace the comments with the actual code)
```jsx
// index.js
import './disable-speedy'
// ...other imports
// ...mounting and stuff

// disable-speedy.js
import { sheet } from 'emotion'
// ...disable speedy
```